mesonbuild/interpreter.py

```diff
@@ -2319,6 +2319,12 @@ def func_configure_file(self, node, args, kwargs):
             raise InterpreterException('Must not specify both "configuration" '
                                        'and "command" keyword arguments since '
                                        'they are mutually exclusive.')
+        if 'capture' in kwargs:
+            if not isinstance(kwargs['capture'], bool):
+                raise InterpreterException('"capture" keyword must be a boolean.')
+            if not 'command' in kwargs:
```

Comment:
__[flake8]__
* [E713] test for membership should be 'not in'

<!-- Comment by [SideCI](https://sideci.com) -->